README.rst

```diff
@@ -5,52 +5,132 @@ plone.staticresources
 Static JS and CSS resources for Plone.
 
 
-This add on contains all JavaScript and CSS resources from Plone.
-Prior to Plone 5.2, they were located in Products.CMFPlone.
+This add on contains all JavaScript and CSS resources used by Plone.
```

Comment:
Note: the main css compilations by default come from barceloneta.
I know that this are different resources but maybe the sentence is not clear enough.